lib/date_time.ex

```diff
@@ -72,17 +72,17 @@ defmodule DateTimeParser.DateTime do
     )
   end
 
-  defp timezone_from_tokens(tokens) do
+  defp timezone_from_tokens(tokens, naive_datetime) do
     with zone <- format_token(tokens, :zone_abbr),
          offset <- format_token(tokens, :utc_offset),
          true <- Enum.any?([zone, offset]) do
-      Timex.Timezone.get(offset || zone)
+      Timex.Timezone.get(offset || zone, naive_datetime)
```

Comment:
To make sure I understand the PR, is this the line that fixes things? 

Based on the commit and docs, it sounds like `Timex.Timezone.get/1` will default the second argument (the timezone) to the system's current time ("if one is not provided, then the current date and time is returned"). This means that if you are parsing a date that is in saving time but running the test on a date that is not in saving time, there would be a discrepancy (and vice versa). 

Instead, this is now passed in the timestamp being parsed, and that will be used as a part of returning timezone info (as opposed to being based on when the test is run).

Reply:
right!